### PR TITLE
Sprints/platform17 3/4

### PR DESCRIPTION
**Pull request by JonathanThomassin**

### CODE REVIEW

File: src/Compliance.Solution/Solution/Entities/Report/SavedQueries/{204ed66b-8bc9-e411-af58-005056900d4b}.xml [hunk 1]
```diff
@@ -1,45 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (GabrielFG) [inline src/Compliance.Solution/Solution/Entities/Report/SavedQueries/{204ed66b-8bc9-e411-af58-005056900d4b}.xml:1]: A whole lot of views have been removed for Reports, is this intentional?

Comment (GabrielFG) [reply src/Compliance.Solution/Solution/Entities/Report/SavedQueries/{204ed66b-8bc9-e411-af58-005056900d4b}.xml:1]: We'll go ahead with the deletions we're not sure how useful they will be in the new system, if we need them back, we can reintroduce them.

File: src/Compliance.Solution/Solution/Entities/SharePointDocument/Entity.xml [hunk 2]
```diff
@@ -1,45 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (GabrielFG) [inline src/Compliance.Solution/Solution/Entities/SharePointDocument/Entity.xml:1]: As discussed, we'll want to make sure if removing this won't have consequences to the fix we have done for the "folder up" SharePoint grid feature.

File: src/Compliance.Solution/Solution/Entities/opc_pafile/FormXml/main/{e1d2d24d-402c-4df8-8476-f592ffa5753b}.xml [hunk 3]
```diff
@@ -1973,27 +1973,62 @@
...
+            <Titles>
+              <Title LCID="1033" Text="Subject Types" />
+            </Titles>
+          </NavBarByRelationshipItem>
+          <NavBarByRelationshipItem RelationshipName="opc_pafile_opc_legalconsultation" Area="Info" Id="nav_opc_pafile_opc_legalconsultation" Sequence="10008">
+            <Titles>
+              <Title LCID="1033" Text="Legal Consultations" />
+            </Titles>
+          </NavBarByRelationshipItem>
+          <NavBarByRelationshipItem RelationshipName="opc_pafile_opc_papipedastatushistory" Area="Info" Id="nav_opc_pafile_opc_papipedastatushistory" Sequence="10009" Show="false">
```
Comment (GabrielFG) [inline src/Compliance.Solution/Solution/Entities/opc_pafile/FormXml/main/{e1d2d24d-402c-4df8-8476-f592ffa5753b}.xml:2016]: Interestingly, even with opc_papipedastatushistory deleted, this nav bar item still exists. Don't think it's the only remaining references either.

File: src/Compliance.Solution/Solution/Other/Relationships/opc_RiskAssessment.xml [hunk 4]
```diff
@@ -18,7 +18,7 @@
     <RelationshipDescription>
       <Descriptions>
         <Description description="Reference to the Risk Assessment." languagecode="1033" />
-        <Description description="Référence à l’évaluation des risques." languagecode="1036" />
+        <Description description="Renvoi à l’évaluation des risques." languagecode="1036" />
```
Comment (JonathanThomassin) [inline src/Compliance.Solution/Solution/Other/Relationships/opc_RiskAssessment.xml:21]: Should not have changed.

File: src/Compliance.Solution/Solution/Entities/opc_pipedafile/FormXml/main/{e44223e0-9cac-4344-b608-82e51cc2e4d0}.xml [hunk 5]
```diff
@@ -2022,7 +2022,13 @@
         </rows>
       </footer>
       <Navigation>
-        <NavBar />
+        <NavBar>
+          <NavBarByRelationshipItem RelationshipName="opc_pipedafile_opc_papipedastatushistory" Sequence="10000" Area="Info" Id="nav_opc_pipedafile_opc_papipedastatushistory" Show="false">
```
Comment (GabrielFG) [inline src/Compliance.Solution/Solution/Entities/opc_pipedafile/FormXml/main/{e44223e0-9cac-4344-b608-82e51cc2e4d0}.xml:2026]: More opc_papipedastatushistory to be removed I believe.

File: src/Compliance.Solution/Solution/SdkMessageProcessingSteps/{5d2b96a3-8b23-4ac9-8ad3-f20e7aeca9ea}.xml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<SdkMessageProcessingStep Name="Components.Support.TagRegistrationPlugin: Create of any Entity" SdkMessageProcessingStepId="{5d2b96a3-8b23-4ac9-8ad3-f20e7aeca9ea}" unmodified="1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <SdkMessageId>9ebdbb1b-ea3e-db11-86a7-000a3a5473e8</SdkMessageId>
+  <PluginTypeName>Components.Support.TagRegistrationPlugin, Components.Support, Version=1.0.0.0, Culture=neutral, PublicKeyToken=2d8165c36fd35df6</PluginTypeName>
+  <PluginTypeId>bba6e816-e1f4-43a9-8348-2e076f0fdd3d</PluginTypeId>
+</SdkMessageProcessingStep>
```
Comment (JonathanThomassin) [inline src/Compliance.Solution/Solution/SdkMessageProcessingSteps/{5d2b96a3-8b23-4ac9-8ad3-f20e7aeca9ea}.xml:6]: This is from the PCF project and should not be in this solution.

File: src/Compliance.Solution/Solution/Other/Solution.xml [hunk 7]
```diff
@@ -534,74 +519,102 @@
...
+        <Dependent type="80" schemaName="opc_ci2" displayName="Ci2" />
+      </MissingDependency>
+      <MissingDependency>
+        <Required type="26" schemaName="Inquires Reports" displayName="Inquires Reports" parentSchemaName="report" parentDisplayName="Report" solution="Active" id="{e7b98801-5974-dd11-90e2-0014221ca588}" />
+        <Dependent type="80" schemaName="opc_ci2" displayName="Ci2" />
+      </MissingDependency>
+      <MissingDependency>
+        <Required type="26" schemaName="Ci2 Intake Reports" displayName="Ci2 Intake Reports" parentSchemaName="report" parentDisplayName="Report" solution="Active" id="{f5de7fef-faad-de11-8c7b-0050568c7e74}" />
+        <Dependent type="80" schemaName="opc_ci2" displayName="Ci2" />
+      </MissingDependency>
```
Comment (GabrielFG) [inline src/Compliance.Solution/Solution/Other/Solution.xml:568]: Should we keep these?

File: src/Compliance.Solution/Solution/SdkMessageProcessingSteps/{5d2b96a3-8b23-4ac9-8ad3-f20e7aeca9ea}.xml (hunk 6 above)
Comment (JonathanThomassin) [reply src/Compliance.Solution/Solution/SdkMessageProcessingSteps/{5d2b96a3-8b23-4ac9-8ad3-f20e7aeca9ea}.xml:6]: I removed it from the solution. We will need to unpack.

File: src/Compliance.Solution/Solution/Entities/opc_pafile/FormXml/main/{e1d2d24d-402c-4df8-8476-f592ffa5753b}.xml (hunk 3 above)
Comment (pierluc-ssoucy) [reply src/Compliance.Solution/Solution/Entities/opc_pafile/FormXml/main/{e1d2d24d-402c-4df8-8476-f592ffa5753b}.xml:2016]: looks like we'll have to remove it manually from the xml. does not seem to display in the form editor.

File: src/Compliance.Solution/Solution/Entities/opc_pipedafile/FormXml/main/{e44223e0-9cac-4344-b608-82e51cc2e4d0}.xml (hunk 5 above)
Comment (pierluc-ssoucy) [reply src/Compliance.Solution/Solution/Entities/opc_pipedafile/FormXml/main/{e44223e0-9cac-4344-b608-82e51cc2e4d0}.xml:2026]: looks like we'll have to remove it manually from the xml. does not seem to display in the form editor.

File: src/Compliance.Solution/Solution/Other/Solution.xml (hunk 7 above)
Comment (pierluc-ssoucy) [reply src/Compliance.Solution/Solution/Other/Solution.xml:568]: it looks like we have to bring them back in the solution (at least temporarily)

File: src/Compliance.Solution/Solution/Entities/SharePointDocument/Entity.xml (hunk 2 above)
Comment (pierluc-ssoucy) [reply src/Compliance.Solution/Solution/Entities/SharePointDocument/Entity.xml:1]: I believe @mitchkirady has already verified this, correct?

**Review by pierluc-ssoucy — APPROVED** [review]

**Pull request merged**